smuggle.py

```diff
@@ -148,6 +148,8 @@ def hrs_detection(host, port, path, method, permute_type, content_length_key, te
             target_urls.append(args.url)
 
         for url in target_urls:
+            if "http://" not in url or "https://" not in url:
+                url="https://" + url
```

Comment:
Please refer to the **[url_parser](https://github.com/anshumanpattnaik/http-request-smuggling/blob/main/lib/Utils.py#L73)** helper function because we cannot append the _protocol scheme_ directly to any random string as the URL will be invalid. 

The tool only accepts URLs, and it will rejects any random input string.